src/commands/slow-mode.ts

```diff
@@ -0,0 +1,599 @@
+import { SlowMode, SlowModeStage } from '@/models/slow-mode';
+import handleSlowMode from '@/slow-mode';
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { sendEventLogMessage } from '@/util';
+import { ChannelType, EmbedBuilder, type ChatInputCommandInteraction } from 'discord.js';
+
+async function slowModeHandler(interaction: ChatInputCommandInteraction): Promise<void> {
+	switch (interaction.options.getSubcommandGroup()) {
+		case 'stage': {
+			switch (interaction.options.getSubcommand()) {
+				case 'set':
+					return setStageHandler(interaction);
+				case 'unset':
+					return unsetStageHandler(interaction);
+			}
+			break;
+		}
+		case 'enable': {
+			switch (interaction.options.getSubcommand()) {
+				case 'auto':
+					return enableAutoSlowModeHandler(interaction);
+				case 'static':
+					return enableStaticSlowModeHandler(interaction);
+			}
+			break;
+		}
+		case null: {
+			switch (interaction.options.getSubcommand()) {
+				case 'disable':
+					return disableSlowModeHandler(interaction);
+				case 'stats':
+					return slowModeStatsHandler(interaction);
+			}
+		}
+	}
+
+	throw new Error('Unknown slow mode command');
+}
+
+async function setStageHandler(interaction: ChatInputCommandInteraction): Promise<void> {
+	await interaction.deferReply({ ephemeral: true });
+
+	const channel = interaction.options.getChannel('channel') ?? interaction.channel;
+	if (!channel) {
+		throw new Error('No channel given');
+	}
+
+	if (channel.type !== ChannelType.GuildText) {
+		throw new Error('Slow mode only applies to text channels');
+	}
+
+	const [ slowMode, ] = await SlowMode.findOrCreate({
```

Comment:
I forgot about this last night but the reason why `_` wasn't working here is because only `argsIgnorePattern` is defined. `varsIgnorePattern` also exists and would fix this https://typescript-eslint.io/rules/no-unused-vars/